packit-ci.fmf

```diff
@@ -2,9 +2,12 @@
 
   summary: run keylime e2e tests
 
+  environment:
+    RUST_IMA_EMULATOR: 1
+
   context:
     swtpm: yes
-    agent: python
+    agent: rust
```

Comment:
maybe in a future commit this can be dropped, as there is only one

Reply:
True. The removal would require test updates so better doing that in the future when change is already settled on keylime side.